lib/components/app/responsive-webapp.js

```diff
@@ -97,9 +96,19 @@ class ResponsiveWebapp extends Component {
       // if in mobile mode and from field is not set, use current location as from and recenter map
       if (isMobile() && query.from === null) {
         setLocationToCurrent({ locationType: 'from' }, intl)
-        setMapCenter(map, pt, initZoomOnLocate)
+        setMapCenter(map, pt)
       }
+    } else if (query.from && query.to) {
+      map?.fitBounds([query.from, query.to], {
+        duration: 500,
+        padding: getFitBoundsPadding(map, 0.2)
+      })
```

Comment:
I think something we discussed was if both location fields are filled, set the nearby center to the destination location? Setting the nearby center to the center of the trip is what we were looking to change

Reply:
That happens in a different PR. This PR only handles map panning